storage/innobase/dict/dict0load.cc

```diff
@@ -3196,95 +3196,105 @@ void dict_load_tablespaces_for_upgrade() {
   mutex_exit(&dict_sys->mutex);
 }
 
-
-/***********************************************************************//**
-Loads a table id based on the index id.
-@return true if found */
-static
-bool
-dict_load_table_id_on_index_id(
-/*===========================*/
-        space_index_t           index_id,  /*!< in: index id */
-        table_id_t*             table_id) /*!< out: table id */
+/***********************************************************************/ /**
```

Comment:
Please new style function header comment (no /*****....*****/, no /*=======*/, function arg comments in the header)